modules/mono/glue/GodotSharp/GodotSharp/Core/Color.cs

```diff
@@ -534,7 +536,11 @@ readonly get
         /// <param name="r">The color's red component, typically on the range of 0 to 1.</param>
         /// <param name="g">The color's green component, typically on the range of 0 to 1.</param>
         /// <param name="b">The color's blue component, typically on the range of 0 to 1.</param>
-        /// <param name="a">The color's alpha (transparency) value, typically on the range of 0 to 1. Default: 1.</param>
+        /// <param name="a">
+		/// The color's alpha (opacity) value, typically on the range of 0 to 1.
+		/// A value of 0 means that the color is fully transparent.
+		/// A value of 1 means that the color is fully opaque. Default: 1.
```

Comment:
The default value is unnecessary, IntelliSense will already surface the default value of a parameter. Since we're making changes to this documentation we could take the opportunity to remove it.

```suggestion
		/// A value of 1 means that the color is fully opaque.
```